andela/hc-rogue-one
# Ft 156577847 integrate livechat,faq and video

#### What does this PR do?
This PR integrates live chat, frequently asked questions and a video.

#### Description of Tasks to be complete
Currently HealthChecks does not support any direct communication with the users nor does it offer a source of more information like common questions that users may face while using the system nor does its documentation provide insight on getting started with the application.

This PR addresses the issues mentioned above and in order to have HealthChecks improve on the user experience of people using the system, a live chatting system has been integrated into the system using Zendesk.

Since there are many possibilities that any challenges that may be faced by a user while using the system may have been faced by another user and basing on that feedback, a page holding frequently asked questions has been added to the system and in the event that a user's challenge hasn't been addressed, they have the ability to ask the questions that will either be communicated to the user through email or added to the frequently asked questions

To provide the users with better insight on the application use, a video briefly describing the system has been added to the DOC section.

#### How should this be tested manually?
1. RUN python manage.py runserver
2. Click the FAQ link to access the frequently asked questions, administrators can log into the dashboard and view any questions that may have been asked as well as respond to them.
3. Click the Doc link and scroll to video-walk-through to access the video
4. As soon as the application is running a small doc on the right hand side of the application will appear to provide a live chat system

#### Checks



  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`

#### Any pivotal tracker user stories?
#156577847
